src/components/connections/ConnectionView.ts

```diff
@@ -49,7 +49,7 @@ export class ConnectionView extends View {
             title
         })
 
-        await this.tree.setMessageHandler(panel)
+        this.tree.setMessageHandler(panel)

```

Comment:
I'm a little weirded out that this didn't error. Is `await` just a no-op if it didn't return an awaitable? Yuck.

Reply:
Yeah, I only noticed this because my typescript linter highlighted it, saying `await has no effect on this kind of statement`. There may be others. I can track them down as part of this patch if you'd like.